styles.css

```diff
@@ -274,240 +210,347 @@ textarea::placeholder {
-    display: inline-block;
-    font-size: 16px;
-    padding: 12px 24px;
-    margin-left: 5%;
-  }
+.recent-button {
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  padding: 12px 24px;
+  margin-left: 5%;
+}
 
-  .container {
-    background-image: url(./images/placeholder.png);
-    background-repeat: no-repeat;
-    padding-bottom: 50%;
-    margin: 5%;
-    display: grid;
-    height: 250px;
-  }
+.container {
+  background-image: url(./images/placeholder.png);
+  background-repeat: no-repeat;
+  padding-bottom: 50%;
+  margin: 5%;
+  display: grid;
+  height: 250px;
+}
 
-  .project-tittle {
-    color: #fff;
-    font-family: 'Crete Round', serif;
-    font-weight: 400;
-    font-size: 32px;
-    line-height: 44px;
-    margin-top: 25%;
-    margin-left: 5%;
-    width: 295px;
-    height: 81px;
-  }
+.project-tittle {
+  color: #fff;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  margin-top: 25%;
+  margin-left: 5%;
+  width: 295px;
+  height: 81px;
+}
 
-  .project-description {
-    color: #fff;
-    font-family: 'Inter', sans-serif;
-    font-style: normal;
-    font-weight: 400;
-    font-size: 16px;
-    line-height: 24px;
-    margin-left: 5%;
-    margin-right: 5%;
-    margin-top: -10px;
-  }
+.project-description {
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin-left: 5%;
+  margin-right: 5%;
+  margin-top: -10px;
+}
 
-  .project-tools {
-    display: flex;
-    justify-content: flex-start;
-    list-style-type: none;
-    margin-left: -20px;
-    font-family: 'Inter', sans-serif;
-  }
+.project-tools {
+  display: flex;
+  justify-content: flex-start;
+  list-style-type: none;
+  margin-left: -20px;
+  font-family: 'Inter', sans-serif;
+}
 
-  .project-tools li {
-    border: 1px solid black;
-    padding: 8px;
-    margin-right: 10px;
-    color: #fff;
-    background: rgba(255, 255, 255, 0.24);
-  }
+.project-tools li {
+  border: 1px solid black;
+  padding: 8px;
+  margin-right: 10px;
+  color: #fff;
+  background: rgba(255, 255, 255, 0.24);
+}
 
-  .project-button {
-    background-color: #ff6b00;
-    color: #fff;
-    border: none;
-    text-decoration: none;
-    display: inline-block;
-    font-size: 16px;
-    height: 48px;
-    width: 327px;
-    text-align: center;
-    margin-bottom: -100px;
-  }
+.project-button {
+  background-color: #ff6b00;
+  color: #fff;
+  border: none;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  height: 48px;
+  width: 327px;
+  text-align: center;
+  margin-bottom: -100px;
+}
 
-  form {
-    margin: 0 auto;
-    padding-top: 20%;
-    padding-bottom: 20%;
-    background-color: #fff;
-  }
+form {
+  margin: 0 auto;
+  padding-top: 20%;
+  padding-bottom: 20%;
+  background-color: #fff;
+}
```

Comment:
- [ ]  Kindly consider laying out the form elements using flexbox as it is a project requirement.
  -  In order to lay out the elements on the page you should use Flexbox.